frontend/src/pages/notebookController/NotebookLogoutRedirect.tsx

```diff
@@ -54,25 +39,13 @@ const NotebookLogoutRedirect: React.FC = () => {
 
   React.useEffect(() => {
     if (namespace && notebookName && namespace !== workbenchNamespace) {
-      if (error) {
-        notification.error(`Error when logging out ${notebookName}`, error.message);
-        navigate(`/projects/${namespace}`);
-      }
-      if (loaded && notebookRoute) {
+      if (notebookRoute) {
         const location = new URL(notebookRoute);
+        // TODO: how do we handle logout? We share a token login state
         window.location.href = `${location.origin}/oauth/sign_out`;
```

Comment:
@lphiri Do we have a path forward for what we are doing with logout? Hopefully the Notebook just doesn't have the action -- but if it does, we will invoke an oauth signout path that is just not going to work and we will get a 404 page likely.